### ISSUE TITLE
[enc2] Undefined variable: newPosition at core\/lib\/private\/files\/stream\/encryption.php#377 …

### ISSUE THREAD

**Issue by DeepDiver1975** [owncloud/core#15973]
````
{"reqId":"ZtZvFotw+V0BNtAyrLmF","remoteAddr":"::1","app":"PHP","message":"Undefined variable: newPosition at \/home\/deepdiver\/Development\/ownCloud\/core\/lib\/private\/files\/stream\/encryption.php#377","level":3,"time":"2015-04-30T12:00:47+00:00","method":"PUT","url":"\/oc\/core\/remote.php\/webdav\/t1-20150430140039-78\/share_source\/foobar.txt"}
{"reqId":"ZtZvFotw+V0BNtAyrLmF","remoteAddr":"::1","app":"PHP","message":"Undefined variable: newPosition at \/home\/deepdiver\/Development\/ownCloud\/core\/lib\/private\/files\/stream\/encryption.php#385","level":3,"time":"2015-04-30T12:00:47+00:00","method":"PUT","url":"\/oc\/core\/remote.php\/webdav\/t1-20150430140039-78\/share_source\/foobar.txt"}
{"reqId":"ZtZvFotw+V0BNtAyrLmF","remoteAddr":"::1","app":"PHP","message":"Undefined variable: newPosition at \/home\/deepdiver\/Development\/ownCloud\/core\/lib\/private\/files\/stream\/encryption.php#377","level":3,"time":"2015-04-30T12:00:47+00:00","method":"PUT","url":"\/oc\/core\/remote.php\/webdav\/t1-20150430140039-78\/share_source\/foobar.txt"}
{"reqId":"ZtZvFotw+V0BNtAyrLmF","remoteAddr":"::1","app":"PHP","message":"Undefined variable: newPosition at \/home\/deepdiver\/Development\/ownCloud\/core\/lib\/private\/files\/stream\/encryption.php#385","level":3,"time":"2015-04-30T12:00:47+00:00","method":"PUT","url":"\/oc\/core\/remote.php\/webdav\/t1-20150430140039-78\/share_source\/foobar.txt"}

**Comment by DeepDiver1975** [comment]
@schiesbn @th3fallen THX

**Comment by DeepDiver1975** [comment]
We basically have to decide what to do in these 3 cases where the if conditions are not met: https://github.com/owncloud/core/blob/master/lib/private/files/stream/encryption.php#L358

**Comment by DeepDiver1975** [comment]
@schiesbn @icewind1991 @PVince81 @jknockaert

**Comment by jknockaert** [comment]
That's an obvious omission. I will come up with a fix later today.

**Comment by jknockaert** [comment]
@DeepDiver1975 Do you actually have an idea of the specific values of the seek parameters in this case?

**Comment by jknockaert** [comment]
Fix proposed in #15980.

**Comment by karlitschek** [comment]
@schiesbn @DeepDiver1975 What can we do to move forward here?

**Comment by jknockaert** [comment]
@karlitschek A fix is ready for review in #15980.
